database/connection.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = require('bluebird');
+
+mongoose.connect('mongodb://localhost/ReviewsDb', { useMongoClient: true });
+
+const ReviewsDb = mongoose.connection;
+
+// Bind connection to error event (to get notification of connection errors)
+ReviewsDb.on('error', console.error.bind(console, 'MongoDB connection error:'));
+
+
+// mongoose.connection.on('connected', function() {
+//   console.log(connected("Mongoose default connection is open to ", dbURL));
+// });
```

Comment:
Is this still needed or can be removed?

Reply:
do you mean the comment?

Reply:
Yah, wasn't sure if you were planning on using it down the line or just leaving it in there for reference. Made a few comments on some commented out code etc, feel free to ignore if you're just keeping it around while you're working on stuff.

Reply:
Makes sense no worries just leftover fodder lol